tests/_data/imports.txt

```diff
@@ -0,0 +1,3 @@
+pandas==2.0.0
```

Comment:
this input files commit could be squashed into the next one. It all relates to the same unit test, I think it would be cleaner to have a single commit for it